utils/jwt.js

```diff
@@ -0,0 +1,29 @@
+const jwt = require('jsonwebtoken');
+
+const createJWT = ({ payload }) => {
+	const token = jwt.sign(payload, process.env.JWT_SECRET, {
```

Comment:
The variable in both '.env' files is called `JWT_KEY`, not `JWT_SECRET`. Devil is in the details.